.github/workflows/comment-pr-on-check-failure.yml

```diff
@@ -37,7 +39,9 @@ jobs:
             });
             var fs = require('fs');
             fs.writeFileSync('${{github.workspace}}/pr.zip', Buffer.from(download.data));
-      - run: unzip pr.zip
+      - name: Unzip the pr zip file
+        if: ${{ github.event.workflow_run.conclusion == 'failure' }}
```

Comment:
remove this we always want to have the pr number at hand

Reply:
@quentinDERORY - the pr.zip file is getting created only on failures from the other check failure workflows

Reply:
> remove the if, we always want to have the pr number at hand

Addressed this comment to propogate PR number on both failures and success. Puushed the changes.